dis_snek/models/snek/application_commands.py

```diff
@@ -922,14 +890,12 @@ def squash_subcommand(subcommands: List) -> Dict:
         sub_cmds = []
         for subcommand in subcommands:
             if not output_data:
-                perms = [s.to_dict() if not isinstance(s, dict) else s for s in subcommand.permissions]
-                perms_filtered = [dict(t) for t in {tuple(d.items()) for d in perms}]
                 output_data = {
                     "name": str(subcommand.name),
                     "description": str(subcommand.description),
                     "options": [],
-                    "permissions": perms_filtered,
-                    "default_permission": subcommand.default_permission,
+                    "default_member_permissions": str(int(subcommand.default_member_permissions)),
```

Comment:
This will raise a `TypeError` for any command that has this set to None. 
![image](https://user-images.githubusercontent.com/22540825/165904508-3a60fa12-efcb-438b-929c-26a927cb1d34.png)
Replication steps: Load debug scale -> attempt to sync